examples/paraToParaParachainPrimaryNativeReserveTransfer.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * When importing from @substrate/asset-transfer-api it would look like the following
+ *
+ * import { AssetTransferApi, constructApiPromise } from '@substrate/asset-transfer-api'
+ */
+import { AssetTransferApi, constructApiPromise } from '../src';
+import { TxResult } from '../src/types';
+import { GREEN, PURPLE, RESET } from './colors';
+
+/**
+ * In this example we are creating a call to send PHA from a Phala Network (Parachain) account
+ * to a Moonbeam (Parachain) account, where the `xcmVersion` is set to 3, and `isLimited` is false declaring that
+ * it will allow `unlimited` weight for the tx.
```

Comment:
This is kinda true, if isLimited is `true` it will be unlimited unless `weightLimit` is provided

Reply:
If isLimited is false it moreso means that it will use `reserveTransferAssets` or `teleport` instead of `limited*`

Reply:
good point! Ive updated these to match the way we describe it in other examples to make it more consistent